examples/oink-oink/src/components/Game/Game.tsx

```diff
@@ -28,7 +28,9 @@ export const Game = memo(() => {
 
   const prevLatestGuessRef = useRef(latestGuess)
   useEffect(() => {
-    if (latestGuess !== prevLatestGuessRef.current) {
+    if (
+      JSON.stringify(latestGuess) !== JSON.stringify(prevLatestGuessRef.current)
+    ) {
```

Comment:
During acting, after 10s `SkipTurnButton` is shown. It is done by modifying game state using `game.currentTurn.showSkipTurnButton = true`.

The change of `game.currentTurn.showSkipTurnButton` causes `latestGuess !== prevLatestGuessRef.current` to 
be true even though the content of these two objects are the same.

Looks like `jotai` does deep copy of the state.
Hence, I am using a simple deep compare here instead of comparing by reference.